README.md

```diff
@@ -250,6 +251,15 @@ Pulls and adds the WWDR Apple Certificate to the given keychain
 | `keychain_name` <br/> `KEYCHAIN_NAME` | The keychain name to install the certificates to | `login` |
 | `keychain_password` <br/> `KEYCHAIN_PASSWORD` | The keychain password to install the certificates to | |
 
+### `is_build_necessary`
+Defines if the build is necessary by looking up changes since the last tag for the same build configuration.<br/>
+If the number of revisions between the last tag matching the given build configuration, and HEAD is higher than 0, the action will return true.
```

Comment:
```suggestion
If the number of revisions between the last tag match the given build configuration, and HEAD is higher than 0, the action will return true.
```

Reply:
Actually I think it's the last tag that matches the given build configuration. Not simply the last tag. What do you think ?

Reply:
Just noticed this message, that does sound like the correct syntax for what is happening ☝️